client.go

```diff
@@ -423,3 +427,33 @@ func (c *Client) String() string {
 func (c *Client) setHeader(key, val string) {
 	c.headers[key] = val
 }
+
+// FeedArgs optional arguments for [fauna.Client.Feed]
+type FeedArgs struct {
```

Comment:
in our call, didn't we land on variadic option functions for consistency? @erickpintor 

i'm still suffering from jet lag, so maybe i'm wrong

Reply:
We did. That was my understanding.

Reply:
sorry about that, we're back to using variadic opt funcs again